packages/components/src/presentation-components/instance-filter-builder/PresentationInstanceFilterDialog.tsx

```diff
@@ -31,9 +31,13 @@ export interface PresentationInstanceFilterDialogProps {
   /** Specifies whether dialog is open or not. */
   isOpen: boolean;
   /** Callback that is invoked when 'Apply' button is clicked. */
-  onApply: (filter: PresentationInstanceFilterInfo) => void;
+  onApply: (filter?: PresentationInstanceFilterInfo) => void;
   /** Callback that is invoked when 'Close' button is clicked or dialog is closed. */
   onClose: () => void;
+  /** Callback that is invoked when 'Reset' button is clicked. */
+  onReset: () => void;
```

Comment:
I kept it obligatory for the sake of keeping all of the callbacks the same. Also, it does nothing because of our implementation and it only so happens to be that we use the other two. Maybe, it would be a good idea to make them all optional. What do you think?

Reply:
I think `onClose` can be made optional too. I don't think it makes sense to use this dialog without `onApply` so I would keep it required.

Reply:
By all, I mean the three button callbacks `onReset`, `onApply` and `onClose`